robocop/checkers/errors.py

```diff
@@ -205,6 +205,21 @@
         severity=RuleSeverity.ERROR,
         version=">=4.0",
     ),
+    "0414": Rule(
+        rule_id="0414",
+        name="duplicated-setting",
+        msg="{{ error_msg }}",
+        severity=RuleSeverity.ERROR,
+        docs="""
+        Some settings can be only used once in file. Only the first value is used.
```

Comment:
Please also update tests
```suggestion
        Some settings can be used only once in a file. Only the first value is used.
```